test/BatchExchange.spec.ts

```diff
@@ -166,9 +159,7 @@ contract("BatchExchange", async (accounts) => {
       );
     });
     it("Burns 10 OWL when adding token", async () => {
-      const TokenOWLProxy = artifacts.require(
-        "../node_modules/@gnosis.pm/owl-token/build/contracts/TokenOWLProxy",
-      );
+      const TokenOWLProxy = artifacts.require("TokenOWLProxy");
```

Comment:
Weird, I hadn't even noticed this was a thing. What was making the OWL token proxy so special before?

Reply:
Nothing, Truffle works with both methods. The main difference is that when you do it this way it works better with the generated types.